src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -32,28 +30,23 @@
 
 class PowerDistributingActor(Actor):
     # pylint: disable=too-many-instance-attributes
-    """Actor to distribute the power between batteries in a microgrid.
+    """Actor to distribute the power between components in a microgrid.
 
-    The purpose of this tool is to keep an equal SoC level in all batteries.
-    The PowerDistributingActor can have many concurrent users which at this time
-    need to be known at construction time.
+    The actor can have many concurrent component managers.
+    For each component manager a bidirectional channel needs to be created
+    through which they can send and receive requests and responses.
 
-    For each user a bidirectional channel needs to be created through which
-    they can send and receive requests and responses.
-
-    It is recommended to wait for PowerDistributingActor output with timeout. Otherwise if
-    the processing function fails then the response will never come.
+    It is recommended to wait for PowerDistributingActor output with timeout.
+    Otherwise if the processing function fails then the response will never come.
```

Comment:
I think these lines can go away, this was a quirk of bidirectional channels.  Now the request and results are decoupled, and results are directly available to users, but users don't send direct requests anymore.

Reply:
gone